src/main/java/frc/robot/utilities/pid/PIDFTalonFX.java

```diff
@@ -43,12 +52,18 @@ public double getTuningSetpoint() {
     }
 
     @Override
-    public void setSetpoint(double setpoint) {
+    public void setTuningSetpoint(double setpoint) {
         tuningSetpoint = setpoint;
+        setWithF(controlMode, setpoint);
```

Comment:
why would setting the setpoint also move the motor?

Reply:
because the setpoint is for the motor...